web/src/main/java/com/futurewei/alcor/web/entity/NetworkVxlanType.java

```diff
@@ -0,0 +1,34 @@
+package com.futurewei.alcor.web.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+@Data
+public class NetworkVxlanType {
+
+    @JsonProperty("id")
+    private String id;
+
+    @JsonProperty("name")
+    private String name;
+
+    @JsonProperty("mtu")
+    private String mtu;
+
+    @JsonProperty("vlan_id")
```

Comment:
This class is for VxLan, so the id should be "vxlan_id", instead of "vlan_id" :-)

Reply:
I think this property is used to store the vlan_id related to vxlan ~ let us talk about it

Reply:
I don't think we discussed this before.